### DISCUSSION TITLE
Training script fonts are different from fonts used in training dataset

### DISCUSSION THREAD

**Discussion by khawar-islam** [mindee/doctr#1037] [Q&A] [answered]
When i have generated a synthetic training dataset, I have used different fonts style contained from different `english` and `korean` like below:

**Training dataset fonts**
![image](https://user-images.githubusercontent.com/11488932/187877362-0404a4ef-56f7-48c4-898e-23673b110709.png)

During training, the script is using different fonts like `FreeMono.ttf,FreeSans.ttf,FreeSerif.ttf`. 

`Namespace(amp=False, arch='master', batch_size=16, device=None, epochs=50, find_lr=False, font='FreeMono.ttf,FreeSans.ttf,FreeSerif.ttf', input_size=32, lr=0.001, max_chars=12, min_chars=1, name=None, pretrained=False, push_to_hub=False, resume=None, sched='cosine', show_samples=False, test_only=False, train_path='/media/cvpr/CM_1/CC/dataset_KR/dataset/train', train_samples=1000, val_path='/media/cvpr/CM_1/CC/dataset_KR/dataset/val', val_samples=20, vocab='korean', wb=False, weight_decay=0, workers=None)`

Are there any problem while training master model? Is this right way?

**Comment by felixdittrich92** [comment, marked as answer]
Hi @khawar-islam you need to pass the list of fonts you want to use as arg to the training script (be aware that the fonts are supported by the vocab characters) 

```
parser.add_argument(
        "--font", type=str, default="FreeMono.ttf,FreeSans.ttf,FreeSerif.ttf", help="Font family to be used"
    )
```
